spec/test_app_templates/lib/generators/test_app_generator.rb

```diff
@@ -11,6 +11,40 @@ def create_generic_work
     generate 'hyrax:work GenericWork'
   end
 
+  def create_nested_work
+    generate 'hyrax:work NestedWork'
+    gsub_file 'app/models/nested_work.rb',
```

Comment:
Given the problems around including in the proper order, feel free to ignore this suggestion.